### PR TITLE
0.3.0: Upgrade redis to 0.25

### PR DESCRIPTION
**Pull request by 5Dev24**
Small upgrade to use redis 0.25

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "redis-queue"
-version = "0.2.0"
+version = "0.2.1"
```
Comment (DoumanAsh) [inline Cargo.toml:3]: Please bump to 0.3.0 as redis change maybe big change

File: tests/manager.rs [hunk 2]
```diff
@@ -82,7 +82,7 @@ async fn create_queue(suffix: &str) -> Queue {
     let stream = format!("stream{suffix}").into();
     let config = QueueConfig { stream };
     let client = redis::Client::open("redis://127.0.0.1/").expect("to create redis client");
-    let conn = client.get_tokio_connection_manager().await.expect("to get connection");
```
Comment (DoumanAsh) [inline tests/manager.rs:85]: why you changed it?

Comment (5Dev24) [reply tests/manager.rs:85]: get_tokio_connection_manager is now deprecated

File: Cargo.toml (hunk 1 above)
Comment (5Dev24) [reply Cargo.toml:3]: Will do

**Review by DoumanAsh — APPROVED** [review]

**Pull request merged**